action/protocol/context.go

```diff
@@ -122,6 +122,7 @@ type (
 		SuicideTxLogMismatchPanic               bool
 		PanicUnrecoverableError                 bool
 		CandidateIdentifiedByOwner              bool
+		ExtendCandidateDelegatesTo48            bool
```

Comment:
what if we use `FeatureWithHeightCtx` to return a `uint64` given a height. such that we can put `height: number` into config as a map, with `genesis.NumCandidateDelegates` as the default value.

Reply:
it seems we need this feature for some purpose, which is ResizeCandidateDelegates (enlarge or reduce). So we may need a better name for the feature.